Zend/tests/property_hooks/backed_delegated_read_wirte.phpt

```diff
@@ -1,5 +1,9 @@
 --TEST--
 Attempted read/write of backing value in a delegated method throws
+--SKIPIF--
+<?php
+if (getenv('SKIP_ASAN')) die('skip FIXME Tweak stack size');
```

Comment:
I will check if I can fix stack limit for ASAN

Reply:
It fails for me locally, but I also have issues with the existing zend_tests.